src/helpViewer/index.ts

```diff
@@ -174,7 +174,11 @@ export class RHelp implements api.HelpPanel, vscode.WebviewPanelSerializer<strin
 	constructor(options: HelpOptions){
 		this.webviewScriptFile = vscode.Uri.file(options.webviewScriptPath);
 		this.webviewStyleFile = vscode.Uri.file(options.webviewStylePath);
-		this.helpProvider = new HelpProvider(options);
+		const pkgListener = () => {
+			void vscode.window.showInformationMessage('Restarting Help Server...');
```

Comment:
When a number of packages are installed, this message appears, but it lasts too long and becomes annoying. I suggest that we only write logging instead of showing a message here.

Reply:
Also, when we update all packages, the library is frequently updated and the help server is restarted many times. The message could be annoying in this case too.